``Development``: Add theming and color usage guidelines (#5116)
ls1intum/Artemis: .github/PULL_REQUEST_TEMPLATE.md

- [ ] I documented the Java code using JavaDoc style.
#### Client
- [ ] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).

> - [ ] Following the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I did not specify colors anywhere other than the theming variable files and checked that the changes look consistent in both the light and the dark theme.

Reviewer: Sorry to break the approval streak, but I think this sentence get simplified a bit:
```suggestion
- [ ] Following the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
```

> - [ ] According to the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I did not specify colors anywhere other than the theming variable files and checked that the changes look consistent in both the light and the dark theme.

Reviewer: ```suggestion
- [ ] Following the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I did not specify colors anywhere other than the theming variable files and checked that the changes look consistent in both the light and the dark theme.
```
